App/Containers/PhotoViewerScreen.js

```diff
@@ -23,6 +25,15 @@ class PhotoViewerScreen extends React.PureComponent {
     this.props.screenProps.dismiss()
   }
 
+  renderImage(props, dims) {
```

Comment:
There was a version of this from back when we were rendering base64 data that passed more of the props/dims into the Image being returned... maybe a good idea to resurrect some of that code?

Reply:
taking a look now